src/In.ProjectEKA.HipService/MessagingQueue/MessagingQueueManager.cs

```diff
@@ -0,0 +1,47 @@
+
+namespace In.ProjectEKA.HipService.MessagingQueue
+{
+    using System;
+    using System.Text;
+    using Microsoft.Extensions.ObjectPool;
+    using Newtonsoft.Json;
+    using RabbitMQ.Client;
+    
+    public class MessagingQueueManager : IMessagingQueueManager
+    {
+        private readonly DefaultObjectPool<IModel> objectPool;  
+        
+        public MessagingQueueManager(IPooledObjectPolicy<IModel> objectPolicy)  
+        {  
+            objectPool = new DefaultObjectPool<IModel>(objectPolicy, Environment.ProcessorCount * 2);  
+        }
+        
+        public void Publish<T>(T message, string exchangeName, string exchangeType, string routeKey) where T : class  
+        {  
+            if (message == null)  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4963271)